clang/test/OpenMP/error_unsupport_feature.c

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -emit-llvm -verify -fopenmp %s
+// RUN: %clang_cc1 -fsyntax-only -verify -fopenmp %s
```

Comment:
It seems like this was a bad idea. When I test this locally this test works for me, but it seems like the it fail in the tests by buildkite/github-pull-requests. I will change this to `-emit-llvm-only` instead an see if that works better.